src/main/java/net/webstructor/self/Siter.java

```diff
@@ -607,8 +609,10 @@ public static int match(Storager storager,String patstr, Iter iter, Thing thing,
 				break;
 			
 			//plain text before "times" and "is" added
-			String nl_text = summary.toString();
-			
+			String nl_text = titler.getMap(path).get(path).toString();
```

Comment:
I don't think that is right. You should add "title" property in addition to "text" property, not replace the "text" with "title", correct?

Reply:
:+1: working on it